Python/Product/EnvironmentsList/PipExtension.xaml.cs

```diff
@@ -201,6 +203,8 @@ PipExtensionProvider provider
             _provider.IsPipInstalledChanged += PipExtensionProvider_IsPipInstalledChanged;
             _provider.InstalledPackagesChanged += PipExtensionProvider_InstalledPackagesChanged;
 
+            IsPipInstalled = _provider.IsPipInstalled ?? true;
```

Comment:
Ok, I don't know if that means we no longer have a way to install pip through that UI, but I think you can always to it by adding a new virtual environment.  In any case, that's not as important as fixing this issue.